pkg/webhooks/validator/validator_test.go

```diff
@@ -1220,6 +1223,43 @@ var _ = Describe("webhooks validator", func() {
 				Entry("should not trigger warning if enableCommonBootImageImport (true) wasn't changed", ptr.To(true), ptr.To(true)),
 				Entry("should not trigger warning if enableCommonBootImageImport (false) wasn't changed", ptr.To(false), ptr.To(false)),
 			)
+
+			//nolint:staticcheck
```

Comment:
I would add a space at the start of the line like other places in the pr

Reply:
same